messages/md.deployreport.json

```diff
@@ -0,0 +1,15 @@
+{
+  "description": "check the status of a metadata deployment",
+  "mdDeployReportCommandCliHelp": "Specify the job ID for the deploy you want to check. You can also specify a wait time (minutes) to check for updates to the deploy status.",
+  "flags": {
```

Comment:
Do we want an example?  If you think it's worth it, you'll also have to wire it up, thx:

"examples": [
    "Check the status of a deploy with job ID 1234 and wait for 10 minutes for the result:",
    "  sfdx force:mdapi:beta:deploy:report -i 1234 -w 100"
  ],

Reply:
adding 2